en/integrations/plugin-packs/procedures/applications-vernemq-restapi.md

```diff
@@ -0,0 +1,168 @@
+---
+id: applications-vernemq-restapi
+title: VerneMQ Restapi
+---
+
+## Overview
+
+VerneMQ is a scalable and open source MQTT broker that connects IoT, M2M, Mobile, and web applications
+The VerneMQ Plugin-Pack monitors Clusters, Listeners, Plugins, and sessions using the RestAPI.
+
+## Plugin-Pack Assets
+
+### Monitored Objects
+
+* VerneMQ including Clusters, Listeners, Plugins, Sessions
+
+### Collected Metrics
+
+More information about collected metrics is available in the official VerneMQ documentation : https://docs.vernemq.com/monitoring/introduction
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--Clusters-->
+
+| Metric name                      | Description                         | Unit  |
+| :------------------------------- | :---------------------------------- | :---- |
+| status                           | Status of clusters                  |       |
+| clusters.running.count           | Number of clusters running          | count |
+| clusters.notrunning.count        | Number of cluster not running       | count |
+
+<!--Listeners-->
+
+| Metric name                      | Description                         | Unit  |
+| :------------------------------- | :---------------------------------- | :---- |
+| status                           | Status of listeners                 |       |
+| listeners.running.count          | Number of listeners running         | count |
+| listeners.notrunning.count       | Number of listeners not running     | count |
+
+<!--Plugins-->
+
+| Metric name                      | Description                         | Unit  |
+| :------------------------------- | :---------------------------------- | :---- |
+| plugins.total.count              | Total number of plugins             | count |
+
+<!--Sessions-->
+
+| Metric name                      | Description                         | Unit  |
+| :------------------------------- | :---------------------------------- | :---- |
+| sessions.online.count            | Number of sessions online           | count |
+| sessions.total.count             | Total number of sessions            | count |
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Prerequisites
+
+A number of distributions provide VerneMQ, including pre-built binary packages.
+Support for these builds, if any, is being provided by the associated distribution vendor.
+Their release cycle may lag behind VerneMQ source releases.
+
+More information is available on the official documentation of VerneMQ : https://docs.vernemq.com/getting-started
+
+The VerneMQ HTTP API is enabled by default and installs an HTTP handler on `http://localhost:8888/api/v1`.
+The centreon-engine user performs a RestAPI connection to this system. 
```

Comment:
RestAPI request OR HTTPx connection, you choose :)

Reply:
RestAPI request it's better